### PR TITLE
heaterFanDemands command

### PR DESCRIPTION
**Pull request by pkubanek**

### CODE REVIEW

File: src/LSST/M1M3/TS/SimulatedFPGA.cpp [hunk 1]
```diff
@@ -222,37 +223,53 @@ void SimulatedFPGA::_simulateModbus(uint16_t* data, size_t length) {
...
-                _heaterPWM[address - 1] = buf.read<uint8_t>();
-                _fanRPM[address - 1] = buf.read<uint8_t>();
-                processThermalStatus(address, 0x20 | 0, 15.0 * random() / float(RAND_MAX),
-                                     _fanRPM[address - 1], 20 * random() / float(RAND_MAX));
-                break;
-            default:
-                SPDLOG_WARN("SimulatedFPGA::_simulateModbus unknown/unsupported function {0:04x} ({0:d})",
-                            func);
+        // broadcasts..
+        if (address >= 248 && address <= 250) {
```
Comment (teweitsai) [inline src/LSST/M1M3/TS/SimulatedFPGA.cpp:227]: Please consider to explain the numbers of 248 and 250.

File: src/LSST/M1M3/TS/SimulatedFPGA.cpp [hunk 2]
```diff
@@ -222,37 +223,53 @@ void SimulatedFPGA::_simulateModbus(uint16_t* data, size_t length) {
...
+            _broadcastCounter = buf.read<uint8_t>();
+            switch (func) {
+                case 88:
```
Comment (teweitsai) [inline src/LSST/M1M3/TS/SimulatedFPGA.cpp:230]: What is 88?

File: src/LSST/M1M3/TS/SimulatedFPGA.cpp [hunk 3]
```diff
@@ -222,37 +223,53 @@ void SimulatedFPGA::_simulateModbus(uint16_t* data, size_t length) {
...
+                                         _fanRPM[address - 1], 20 * random() / float(RAND_MAX));
+                    break;
+                case 89:
+                    processThermalStatus(address, 0x20 | _broadcastCounter, 15.0 * random() / float(RAND_MAX),
+                                         _fanRPM[address - 1], 20 * random() / float(RAND_MAX));
+                    break;
+                default:
+                    SPDLOG_WARN("SimulatedFPGA::_simulateModbus unknown/unsupported function {0:04x} ({0:d})",
+                                func);
+            }
```
Comment (teweitsai) [inline src/LSST/M1M3/TS/SimulatedFPGA.cpp:266]: Please consider to explain the magic numbers such as 18, 65, etc.

Comment (pkubanek) [reply src/LSST/M1M3/TS/SimulatedFPGA.cpp:266]: I added this comment:

// Modbus functions - please see ILC protocol document for details

its better to point yo source of true, then to explain that complicated numbering in code.

**Pull request merged**